packages/semver/src/schematics/ng-add/schema.json

```diff
@@ -13,6 +13,18 @@
     "projects": {
       "description": "Which projects should be versioned independently",
       "type": "array"
+    },
+    "enforceConventionalCommits": {
+      "description": "Install & configure commitlint, husky & commitizen",
+      "type": "boolean",
+      "default": false,
+      "x-prompt": "Enforce conventional commits?"
+    },
+    "skipInstall": {
+      "description": "Skips installing new node_modules after modifying package.json",
+      "type": "boolean",
+      "default": false,
+      "x-prompt": "Skips installing new modules?"
```

Comment:
I don't think we need to prompt for this option.

Reply:
I agree, done